core/core-api/src/main/java/io/fabric8/launcher/core/api/events/StatusEvent.java

```diff
@@ -0,0 +1,15 @@
+package io.fabric8.launcher.core.api.events;
+
+
+/**
+ * Contract for all status messages that we send to the clients via
+ * websockets to inform them about the status of their project.
+ */
+public interface StatusEvent {
```

Comment:
I think it makes sense to name it `StatusEventType`, as it's part of `StatusMessageEvent`

Reply:
9747e0d